metrics/next-ios-render-test-runner-metrics.json

```diff
@@ -0,0 +1,19 @@
+{
+    "base_test_path": "tests",
+    "cache_path": "cache-metrics.db",
+    "expectation_paths": [
+        "expectations/platform-all",
+        "expectations/platform-ios"
+    ],
+    "ignore_paths": [
+        "ignores/platform-all.json",
+        "ignores/platform-macos.json",
+        "ignores/platform-ios.json"
+    ],
+    "metric_path": "next-ios-render-test-runner",
+    "probes": [
+        "probeGFX",
+        "probeMemory",
+        "probeNetwork"
+    ]
```

Comment:
I think these should be removed. Could be the reason for `"probes/gfx/pass": "Needs to be investigated and fixed."` failures.